SphericalShell/OpenMC_Input/OpenMC_Depletion/SS_Depletion.py

```diff
@@ -0,0 +1,53 @@
+import openmc
+import openmc.deplete
+import numpy as np
+import argparse
+import yaml
+
+def deplete_ss(chain_file_path, model, inner_radius, thickness, times_post_boc, particle_source_rates, norm_mode, timestep_units):
+    material = model.materials[0]
+    chain_file = chain_file_path

```

Comment:
swapping these lines keeps all the materials modifications together
```suggestion
    chain_file = chain_file_path
    
    material = model.materials[0]
```